README.md

```diff
@@ -75,6 +75,25 @@ or
 yarn
 ```
 
+Given we have linked the `react-phoenix` javascript dependency locally from your
+Phoenix project's `deps` folder, we should add webpack aliases for react and react-dom
+to your `assets/webpack.config.js`:
```

Comment:
Can you rephrase this paragraph? I'd like to send people on this path only if necessary. Perhaps:

```
#### `npm` users
`npm` unfortunately needs an extra bit of configuration to make it happy. Add the following block to your
`assets/webpack.config.js` file. If you use `yarn` to manage your dependencies, this extra bit of
configuration should not be necessary.
```

Reply:
Sure thing! Your phrasing reads much more nicely - have updated to use it 😄